NorthstarDLL/scriptuserinfo.cpp

```diff
@@ -89,7 +89,7 @@ ADD_SQFUNC("bool", GetUserInfoKVBool_Internal, "entity player, string key, bool
 	"Gets the bool value of a given player's userinfo convar by name", ScriptContext::SERVER)
 // clang-format on
 {
-	const R2::CBasePlayer* pPlayer = g_pSquirrel<ScriptContext::SERVER>->getentity<R2::CBasePlayer>(sqvm, 1);
+	const R2::CBasePlayer* pPlayer = g_pSquirrel<ScriptContext::SERVER>->template getentity<R2::CBasePlayer>(sqvm, 1);

```

Comment:
Not sure where these are `->template` now? How/why does this even work?

Reply:
https://en.cppreference.com/w/cpp/language/dependent_name (The template disambiguator for dependent names) is required for compilation with gcc.